app/views/users/show.html.erb

```diff
@@ -1,16 +1,38 @@
-<div class="show-page">
-<div class="users">
-<%= render 'shared/user', user:@user %>
-</div>
-<div>
-    <%= @user.bio%>
-</div>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <div class='d-flex justify-content-center'>
+    <div class='col-md-4 text-center'>
+      <h2>Sign up</h2>
 
-<div class="posts">
-<h2>Recent Posts</h2>
-<% @recent_posts.each do |post| %>
-<%= render 'shared/post', post:post, user:@user %>
-<% end %>
-</div>
-<%=link_to 'See all posts', user_posts_path(@user) %>
-</div>
+      <div class="form-group">
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email", class:"form-control" %>
+      </div>
+
+      <div class="form-group">
+        <% if @minimum_password_length %>
+        <em>(<%= @minimum_password_length %> characters minimum)</em>
+        <% end %><br />
+        <%= f.password_field :password, autocomplete: "new-password", placeholder: "Password", class:"form-control" %>
+      </div>
+
+      <div class="form-group">
+        <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: "Confirm Password", class:"form-control" %>
+      </div>
+
+      <div class="form-group">
+        <%= f.text_field :name, placeholder: "Names", class:"form-control" %>
+      </div>
+      <div class="form-group">
```

Comment:
- [ ] Great effort working on your project, but currently I am not able to navigate to the user#show page which has hindered me from adding a post or comment for a particular user. this as well as also affected me to navigate through the API routes for posts and comments. Below is a video demonstration of what I am talking about. Thank you.

video 👇 

[screen-capture (2).webm](https://github.com/firstpj/BLOG-appllication/assets/78634841/734a7719-8a5b-4351-b307-e423c6646a8b)